### DISCUSSION TITLE
Pregunta con el sensor de cantidad de viandas

### DISCUSSION THREAD

**Discussion by ParedesMicaela** [dds-jv/foro#439] [TPA - 2024]
Buen día! Estábamos viendo el enunciado y no terminamos de entender qué haría la acción dentro del LSensor. Entendemos que va a ser parecido al sensor de temperatura entonces lo implementamos de manera similar. Pusimos el adapter del sensor y sabemos que probablemente tengamos que hacer un command para la acción (LunchBoxAction) que creemos que es para aperturas de agregarVianda y retirarVianda. No entendemos qué haría la heladera a partir de este cambio en las viandas.

**Comment by flbulgarelli** [comment]
:wave: ¡Buenas!

> Entendemos que va a ser parecido al sensor de temperatura entonces lo implementamos de manera similar.

:+1: 

> hacer un command para la acción (LunchBoxAction) 

:+1: De todas formas, en sentido estricto, no tienen que _diseñar_ ese command, sino implementarlo, para poder procesar el evento del cambio de cantidad de viandas, lo cual les será útil para resolver uno o más requerimientos. 

> No entendemos qué haría la heladera a partir de este cambio en las viandas.

Recuerden que ustedes no están desarrollando el sistema de (cada una de) las heladeras, sino el sistema de gestión de las mismas y sus donaciones. Entonces, no es importante qué hace la heladera cuando cambia la cantidad de viandas, sino que hará el sistema de gestión cuando las mismas cambien (por ejemplo, ¿puede ser relevante que la cantidad de viandas en una heladera pasa a ser cero? ¿deberíamos hacer algo a raiz de ello?.

Saludos
